src/main/java/uk/gov/pay/connector/paymentprocessor/resource/CardResource.java

```diff
@@ -255,6 +255,29 @@ public Response markChargeAsCaptureApproved(@PathParam("accountId") Long account
         return ResponseUtil.noContentResponse();
     }
 
+    @POST
+    @Path("/v1/api/charges/{chargeId}/capture")
+    @Produces(APPLICATION_JSON)
+    @Operation(
+            summary = "Mark delayed capture charge as eligible for capture and adds charge to capture queue",
+            description = "This endpoint should be called to capture a delayed capture charge. The charge needs to have been previously marked as AWAITING CAPTURE REQUEST for this call to succeed. " +
+                    "When a charge is in any of the states CAPTURED, CAPTURE APPROVED, CAPTURE APPROVED RETRY, CAPTURE READY, CAPTURE SUBMITTED then nothing happens and the response will be a 204. " +
+                    "When a charge is in a status that cannot transition (eg. none of the above) then 409 response is returned. ",
+            responses = {
+                    @ApiResponse(responseCode = "204", description = "No content"),
+                    @ApiResponse(responseCode = "409", description = "Conflict - if charge is not in correct state"),
+                    @ApiResponse(responseCode = "404", description = "Not found - charge not found"),
+                    @ApiResponse(responseCode = "500", description = "Internal server error")
+            }
+    )
+    public Response markChargeAsCaptureApproved(@PathParam("chargeId") String chargeId,
```

Comment:
if you renamed this to something like 
```suggestion
    public Response markChargeAsCaptureApprovedByChargeId(@PathParam("chargeId") String chargeId,
```
you can avoid the `_1` in the openapi spec for the operation ID